### PR TITLE
Resolves "Mapping - Add Apps to all TrafficLights"

### PR DESCRIPTION
**Pull request by vogtfa**
## Type of this PR 

- [x] Bug fix
- [ ] Enhancement

## Description

- It is now possible to add multiple apps distributed by weights to traffic lights. If the parameters"weight" and "tlGroupId" in the traffic light mapping, then the default value of the weight is 1. 
- The mapping schema was adapted, so that weights in the traffic light mapping can be bigger than 1

## Issue(s) related to this PR

 * Resolves issue #35
 
 
## Affected parts of the online documentation

Traffic light documentation in docs/simulators/application_simulator/

## Definition of Done

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] All commits are signed-off (`-s`) with your mail address of your Eclipse Account.
- [x] `origin/main` has been merged into your Fork.
- [x] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [x] There are no new SpotBugs warnings. 
- [x] Coding guidelines have been observed (see CONTRIBUTING.md).
- [x] All tests pass.

### CODE REVIEW

File: fed/mosaic-mapping/src/test/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFrameworkTrafficLightTest.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        SpawningFramework spawningFramework = new SpawningFramework(mappingConfig, tlRegistration, rti, rng);
+        spawningFramework.timeAdvance(0, rti, rng);
+
+        //ASSERT
+        verify(rti, times(1000)).triggerInteraction(captor.capture());
+        List <TrafficLightRegistration> tlrs = captor.getAllValues();
+        Assert.assertEquals(countAppInTlr(tlrs, "app_1") / 1000d, 0.2d, 0.02d);
+        Assert.assertEquals(countAppInTlr(tlrs, "app_2") / 1000d, 0.8d, 0.02d);
+        Assert.assertEquals(tlrs.get(13).getMapping().getApplications().get(0), "app_3");
+    }
```
Comment (kschrab) [inline fed/mosaic-mapping/src/test/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFrameworkTrafficLightTest.java:108]: I like that :) thanks! 
However, it would be great to see another test to see an equal distribution in the case `tlGroupId == null && weight == null`

Comment (kschrab) [inline fed/mosaic-mapping/src/test/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFrameworkTrafficLightTest.java:108]: I like that :) thanks! 
However, it would be great to see another test to see an equal distribution in the case `tlGroupId == null && weight == null`

Comment (kschrab) [reply fed/mosaic-mapping/src/test/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFrameworkTrafficLightTest.java:108]: a sorry! I overlooked that it's already there 🙈
